src/map.h

```diff
@@ -666,19 +667,4 @@ static inline bool IsVoxelInsideWorld(const XYZPoint16 &vox)
 	return vox.z >= 0 && vox.z < WORLD_Z_SIZE && IsVoxelstackInsideWorld(vox.x, vox.y);
 }
 
-/**
- * Is both corners of given edge raised of a voxel with the
- * given non-steep slope?
- * @param edge The edge to check
- * @param slope The slope of a voxel.
- * @pre (slope & TSB_STEEP) == 0
- * @return True if both corners of given edge are raised, otherwise false.
- */
-static inline bool IsRaisedEdge(TileEdge edge, TileSlope slope)
```

Comment:
/me wonders if this function is useful elsewhere (or is already "used" elsewhere)